lib/koans/13_functions.ex

```diff
@@ -25,7 +25,7 @@ defmodule Functions do
     assert first("One") == ___
   end
 
-  def repeat_again(message, times \\ 5) do
+  def repeat_again(message, times \\ 3) do
```

Comment:
⚠️ This will cause [a test](https://github.com/elixirkoans/elixir-koans/blob/a2eaef167080a0f3223bbebb8332c00a21d0c7af/test/koans/functions_koans_test.exs#L10) to fail.

Reply:
Fixed! Sorry about that.